.mocharc.js

```diff
@@ -8,8 +8,8 @@ module.exports = {
   extension: [
     'ts'
   ],
-  spec: "test/**/*-test.ts",
-  
+  spec: "test/**/*.test.ts",
```

Comment:
In general, our other projects seem to favor `*.test.ts` files to distinguish test files over `*-test.ts`.

I actually baked this assumption into the `@xpring-eng/eslint-config-base`, so there are a bunch of test-file specific overrides using the `*.test.ts` pattern.

So, I just went ahead and migrated the test files to that naming convention.